hinted-compilation/hinted-mods.py

```diff
@@ -322,20 +322,20 @@ def onModuleEncounter(self, module_filename, module_name, module_kind):
             Example: (False, "because it is not called").
         """
         full_name = module_name
-        elements = full_name.split(".")
+        element = full_name.getTopLevelPackageName()
```

Comment:
The `top_level_package_name` would be way better than `element` in downwards code to understand things, `element` is just meaningless.